database/migrations/model/Version20220211202106.php

```diff
@@ -0,0 +1,61 @@
+<?php namespace Database\Migrations\Model;
+/**
+ * Copyright 2022 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+use LaravelDoctrine\Migrations\Schema\Builder;
+use LaravelDoctrine\Migrations\Schema\Table;
+use LaravelDoctrine\ORM\Facades\Registry;
+use models\summit\Summit;
+use models\utils\SilverstripeBaseModel;
+/**
+ * Class Version20220211202106
+ * @package Database\Migrations\Model
+ */
+class Version20220211202106 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $builder = new Builder($schema);
+        if($schema->hasTable("Summit") && !$builder->hasColumn("Summit", "AllowAutomaticReminderEmails ")) {
+            $builder->table('Summit', function (Table $table) {
+                $table->boolean('AllowAutomaticReminderEmails ')->setNotnull(true)->setDefault(true);
```

Comment:
please rename db field to **RegistrationAllowAutomaticReminderEmails**

Reply:
Done